components/pages/farms/stake/index.tsx

```diff
@@ -26,10 +26,6 @@ export function Stake({ farm }) {
     const [toggle, setToggle] = useState(false)
     const [current, setCurrent] = useState(farm)
     const [lpTokenName, setLpTokenName] = useState('')
-    let [chadEarned, setChadEarned] = useState('')
-
-
-
```

Comment:
chadEarned wasn't used since we are using the "current" object for everything. Also empty lines.